CMakeLists.txt

```
@@ -1,4 +1,5 @@
 option(FORMAT_SKIP_CMAKE "Skip CMake formatting.")
+option(FORMAT_SKIP_CLANG "Skip CMake formatting.")
```

The description should be updated as well.

```suggestion
option(FORMAT_SKIP_CLANG "Skip Clang formatting.")
```

---

Fixed